routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
+const { Widget } = require('../../../models/widgets.js')
+
+const express = require('express')
+const router = express.Router()
+
+router.get('/', (request, response, next) => {
+  Dashboard.find().exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
```

Comment:
Since this is an API, we need to return an appropriate error code for the error that occurred, instead of rendering the error view (which is what will happen by default since this error will get propogated to the error handler in bin/www).  We should probably consider removing the plumbing that express generator added for views since we will no longer be using them... (this applies to all `catch` statements in these files)